data/addFieldsToCollection.js

```diff
@@ -88,6 +109,77 @@ async function setLevelInCourses() {
   });
 }
 
+/* 
+  Script to calculate the post reqs for each sis course.
+*/
+async function setPostReqsInSISCourses() {
+  await db.connect();
+  var counter = 0;
+  SISCV.find({ version: { $elemMatch: { postReq: { $size: 0 } } } }).then(
+    async (res) => {
+      for (let course of res) {
+        let postReqs = [];
+        await SISCV.find({
+          versions: {
+            $elemMatch: {
+              preReq: { $elemMatch: { Expression: { $regex: course.number } } },
+            },
+          },
+        }).then(async (res2) => {
+          for (let matchedCourse of res2) {
+            for (let preReq of matchedCourse.versions[0].preReq) {
```

Comment:
Why are we taking the first course in the versions array?

Reply:
That was for simplificity while testing an assumes that pre req versions of courses don't change very often.
Will update to at least find the latest version, (see previous comment, is pretty related)